### ISSUE TITLE
[PENDING] Support languages other than English

### ISSUE THREAD

**Issue by sergeevabc** [aardappel/procrastitracker#53]
Dear @aardappel,
your application belongs to the works of the 21st century, yet it seems not to be aware of Unicode:

- HTML export is done without UTF8, so we see ```ÐÐµ Ð² ÑÐµÐ±Ðµ``` instead of ```Не в себе```
- filter on sub-string within Statistics View does not work with Russian input (e.g. ```себе```)

Would you mind fixing it?

**Comment by aardappel** [comment]
Fixed filter widget: https://github.com/aardappel/procrastitracker/commit/09dd43db8ddb83e4d5e0cbf8d1bf832c13819d0e

**Comment by sergeevabc** [comment]
Chrome treats HTML without specified encoding as UTF8. Its market share is [~65%][1], but what about the rest? Other browsers need a nudge by means of ```<meta charset="utf-8">``` within ```<head>…</head>```.

[1]: https://netmarketshare.com/browser-market-share.aspx

**Comment by aardappel** [comment]
Also now declare UTF-8 in html export, maybe that will fix it: https://github.com/aardappel/procrastitracker/commit/efee06ffb2b0a664b5cb788b9eb9cb9286d5ef93

**Comment by aardappel** [comment]
And just updated installer on the site again, including these 2 fixes. Let me know if it works for you.

**Comment by sergeevabc** [comment]
Looking forward to verify your fixes, but the app [got broken][1].

[1]: https://github.com/aardappel/procrastitracker/issues/52#issuecomment-455695705

**Comment by aardappel** [comment]
No, it works case insensitive with english text, but again this uses non-unicode aware code. Had a look at what it takes to fix this, and this one is a bit more involved to fix, so may have to wait.

**Comment by sergeevabc** [comment]
Is there anything a mere Windows user like me could do to expedite a resolution of this issue?

**Comment by aardappel** [comment]
This is one of many open source projects I support, and doesn't make me any money. So whether a feature gets implemented depends on whether I or some other programmer want to. If you know someone who could help, that could expedite it.

**Comment by sergeevabc** [comment]
Err… Hello?
